### DISCUSSION TITLE
Historic discovery of the highest redshift supernovae (80!)

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#232] [Astrophysics and Physics]
Supernovae are important for many cosmological tests, especially wether cosmic time dilation exists or not.
But we have always been limited by low redshift supernovae (datasets often cap at Z=1.1). There was only a handful (like 2-3) ~Z=2 supernovae discovered by Hubble, and a few Z=2.5 discovered by the subaru primecam discovered prior and zero via JWST.
I recently stumbled upon the discovery of the highest ever, spectroscopically confirmed sn1a supernova at Z=2.9
https://arxiv.org/abs/2406.05089
However my mind was completely blown away today when I read this new NASA blog post that says over 80 Z>=2 supernovae candidates have been found in the JADES field. Which is ten time more than **all** previously known high Z supernovae, in a single sky survey.
NOT ONLY THAT, but the blog casually drop that:
> The team identified a number of high-redshift supernovae, including the farthest one ever[ spectroscopically](https://webbtelescope.org/glossary.html#h3-CK-4cfac646-5c48-45b7-956a-70caac1af29c) confirmed, at a **redshift of 3.6.**

A supernovae at Z=3.6 is absolutely mind blowing and would revolutionize cosmic time dilation tests. The signal to noise ratio should be improved by ~360% (simplist assumption because reddening and stretch parameter evolve with redshift).
However I just can't find the papers yet (except for the 2.9), it seems it's still in the works?
The hype is real.
https://science.nasa.gov/missions/webb/nasas-webb-opens-new-window-on-supernova-science/
edit not an expert on supernovae but it seems it is a type II supernovae (core collapse) which are less testable but the 2.9 one is a SNIa and therefore still a ~290% increase in signal to noise ratio, especially since no one tested the hubble and subaru ones previously either.

**Comment by RedshiftDrift** [comment]
@LifeIsStrange thanks for the links to articles about high-z supernovae.

I wonder, how do you think these will confirm *whether cosmic time dilation exists or not*? If the analysis is done using $\Lambda\mbox{CDM}$, they will get the same results with high-z SNe since time dilation is part of the analysis.
